bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/form/FormConstants.java

```diff
@@ -109,6 +109,11 @@ private FormConstants() {
     public static final String RT_FD_FRAGMENT_CONTAINER_V1 = RT_FD_FORM_PREFIX + "fragmentcontainer/v1/fragmentcontainer";
 
     public static final String FORM_FIELD_TYPE = "form";
-
     public static final String REQ_ATTR_FORMCONTAINER_PATH = "formContainerPath";
+
+    /** The property name for the additional property keys within the multi field */
+    public static final String PN_CUSTOM_PROPERTY_ADDITIONAL_KEYS = "fd:additionalCustomPropertyKeys";
```

Comment:
These properties should still not be required, you need to directly access it via cq-guides, since cq-guides own these properties